### ISSUE TITLE
Bio link public access gives 404 page not found

### ISSUE THREAD

**Issue by ByteGG** [uilibrary/Linkdrop-Feature-And-Issues#2]
I created a bio link it's accesible when I'm logged in

![image](https://github.com/uilibrary/Linkdrop-Feature-And-Issues/assets/106935571/0f57af35-515d-4b58-8cc6-3f6a1482c745)

Once I try it as none logged in user it give error 404 page not found

![image](https://github.com/uilibrary/Linkdrop-Feature-And-Issues/assets/106935571/15ec7dc4-be42-4c30-83a5-0d07b4a07fb9)

**Comment by ByteGG** [comment]
Issue not resolved and new issue introduced where not able to change the theme from the available themes

**Comment by jamir-hossain** [comment]
It's already fixed. If you use the latest version then you will access biolink from anywhere.

**Comment by ByteGG** [comment]
I tested it again. Not resolved. too many issues not for me but I saw comments from other users too.

I would suggest doing testing in new environment to simulate the installtion for new users.

Giving you access to fix the issue is not suitable for my setup. The issue is clearly from the source code. I have other websites running all working fine including SMTP emal from Amazon SES. The issue is with the script.
